### DISCUSSION TITLE
how to increase the number of epochs?

### DISCUSSION THREAD

**Discussion by kanoyo-git** [voicepaw/so-vits-svc-fork#741] [Q&A] [answered]
I train a model with kaggle (but on colab as well), but trains with a limit of 10000 epochs, is there a parameter (via !) that increases the limit of epochs or what is the way to increase the number of epochs?

**Comment by Grownz** [comment, marked as answer]
Not as a parameter, but in the config file of your training data you can edit all sorts of things (including the count of epochs):
![grafik](https://github.com/voicepaw/so-vits-svc-fork/assets/33704684/5a90d618-9d40-437f-93c3-e49737b8b997)
